src/StackExchange.Redis/ConfigurationOptions.cs

```diff
@@ -313,7 +324,23 @@ private static bool CheckTrustedIssuer(X509Certificate2 certificateToValidate, X
             chain.ChainPolicy.UrlRetrievalTimeout = new TimeSpan(0, 0, 0);
```

Comment:
VerificationTime is being set to DateTime.Now.  This is either redundant to the constructor (which did the same thing), or is a few ticks later.

This is creating a skew problem from when the original request happened (the cert could have exited validity), so you probably want to copy the chain.ChainPolicy.VerificationTime value from the chain you were given in the callback.

Reply:
Done!